src/masoniteorm/orm/query/QueryBuilder.py

```diff
@@ -315,15 +321,26 @@ def create(self, creates={}, query=False, **kwargs):
         """
         if not creates:
             creates = kwargs
+
         self.set_action("insert")
         self._creates.update(creates)
         if query:
             return self
 
-        self.connection.query(self.to_sql(), self._bindings)
-        if self.owner:
-            return self.owner.hydrate(creates)
-        return creates
+        connection = self.new_connection()
+        result = connection.query(self.to_sql(), self._bindings, results=1)
+
+        if self._model:
+            id_key = self._model.get_primary_key()
+
+        processed_results = self.get_processor().process_insert_get_id(
```

Comment:
it took me a bit of time to understand this part, maybe should we name `processed_results` / `results` consistently ?

With results=1 and not * I get that this will return one record, the one just created. So I am okay with `result` as returned value and then after doing post-processing should we keep `processed_result` without s ?

I find it more clear 👍